<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxNotifierProp</class>
 <widget class="QWidget" name="RzxNotifierProp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="grpChat" >
     <property name="title" >
      <string>Chat</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QPushButton" name="btnTestStyleChat" >
        <property name="text" >
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbStyleChat" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblStyleChat" >
        <property name="text" >
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <widget class="QCheckBox" name="cbNoChatForeground" >
        <property name="text" >
         <string>No chat notification when qRezix is the foreground application</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QCheckBox" name="cbNotifyChat" >
        <property name="text" >
         <string>Notify when receive a chat message</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="grpWhat" >
     <property name="title" >
      <string>Global</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbILeave" >
        <property name="text" >
         <string>No notification when I am away</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="grpConnectionState" >
     <property name="title" >
      <string>Connection state</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="3" >
       <widget class="QCheckBox" name="chkBeepFavorites" >
        <property name="whatsThis" >
         <string>&lt;h3>Beep on connection&lt;/h3>&lt;p>Check this if you want qRezix to play a sound when a favorite get connected&lt;/p></string>
        </property>
        <property name="text" >
         <string>Beep on connection of one of your favorites</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>1</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>90</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="txtBeepFavorites" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>&lt;h3>Beep on connection&lt;/h3>&lt;p>Select the sound to play when a favorite get connected&lt;/p></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnBeepBrowse" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="autoRaise" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3" >
       <widget class="QCheckBox" name="cbDisconnect" >
        <property name="text" >
         <string>Notify favorite disconnection</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3" >
       <widget class="QCheckBox" name="cbAway" >
        <property name="text" >
         <string>Notify when favorite go away</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3" >
       <widget class="QCheckBox" name="cbHere" >
        <property name="text" >
         <string>Notify when favorite arrive or come back</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblStyleConnection" >
        <property name="text" >
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbStyleConnection" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="btnTestStyleConnection" >
        <property name="text" >
         <string>Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>564</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
